transformer/cloudfoundrytransformer.go

```diff
@@ -183,16 +200,38 @@ func (t *CloudFoundry) Transform(newArtifacts []transformertypes.Artifact, alrea
 			if serviceContainer.Image == "" {
 				serviceContainer.Image = sConfig.ServiceName
 			}
+			addedAlready := map[string]bool{}
+			// Manifest
 			for varname, value := range application.EnvironmentVariables {
 				serviceContainer.Env = append(serviceContainer.Env, core.EnvVar{Name: varname, Value: value})
+				addedAlready[varname] = true
 			}
 			//TODO: Add support for services, health check, memory
 			if application.Instances.IsSet {
 				serviceConfig.Replicas = application.Instances.Value
 			} else if cfinstanceapp.Application.Instances != 0 {
 				serviceConfig.Replicas = cfinstanceapp.Application.Instances
 			}
+			if cfinstanceapp.Application.Name != "" {
+				var b bytes.Buffer
+				encoder := json.NewEncoder(&b)
+				if err := encoder.Encode(cfinstanceapp.Environment.SystemEnv["VCAP_SERVICES"]); err != nil {
+					logrus.Error("Error while Encoding object to json in cf transform")
+				} else {
+					vcapVariables, err := parseVcapVariables(string(b.Bytes()))
+					if err != nil {
+						logrus.Debugf("Error while parsing vcap variables : %s", err)
+					} else {
+						ir.Storages = append(ir.Storages, irtypes.Storage{Name: config.ServiceName, Content: vcapVariables})
```

Comment:
Do we not want to set the type of storage to secret? We should possibly expose it as a flag and do it only in certain cases.